guides/m1x/magefordev/mage-for-dev-5.html

```diff
@@ -184,7 +184,7 @@ <h3>The Global Config and Creating The Model</h3>
 &lt;/global>
 </pre>
 
-<p>The outer &lt;weblog /&gt; tag is your Group Name, which should match your module name.  &lt;class /&gt; is the BASE name all Models in the weblog group will have, also calles <strong>Class Prefix</strong>. The &lt;resourceModel /&gt; tag indicates which Resource Model that weblog group Models should use.  There's more on this below, but for now be content to know it's your Group Name, followed by a the literal string "resource".</p>
+<p>The outer &lt;weblog /&gt; tag is your Group Name, which should match your module name.  &lt;class /&gt; is the BASE name all Models in the weblog group will have, also calles <strong>Class Prefix</strong>. The &lt;resourceModel /&gt; tag indicates which Resource Model that weblog group Models should use.  There's more on this below, but for now be content to know it's your Group Name, followed by the literal string "resource".</p>
 
 <p>So, we're not done yet, but let's see what happens if we clear our Magento cache and attempt to instantiate a blogpost Model.  In your testModelAction method, use the following code</p>
```

Comment:
Remove double spacing after "Model."

Reply:
testModelAction should be encased in code formatting, like ` `testModelAction` `

Reply:
Please ensure all terms that require code formatting are formatted correctly (with back ticks surround them), like <weblog />, config.xml, <resourceModel />, etc.

Reply:
@shrielenee Removed unnecessary spaces.

Reply:
@shrielenee Added backticks.